simul/platform/MININET.md

```diff
@@ -11,17 +11,17 @@ between servers, but not necessarily between all nodes, Mininet will set up the
 same restriction between any two nodes that communcate with each other.
 
 The Mininet simulation uses http://mininet.org/ to simulate a network where each
-host represents a cothority node. It has been extensively tested on the
+host represents a Cothority node. It has been extensively tested on the
 hardware of https://iccluster.epfl.ch, but should theoretically also run on
 other kinds of hardware. As it needs to set up the mininet environment, it is
 more fragile than the Deterlab simulations. But server availability in EPFL
 is often better than on Deterlab.
 
 Each T3-server on the iccluster is a 24-core server with 256GB of RAM and can
 run around 300 cothority nodes simultaneously. So if you want to run a
-simulation with 2000 nodes, you need at least 7 servers.
+simulation with 2000 nodes, you need at least 7 physical servers.
 
-Unfortunately we still have the restriction that _all servers need to be on the
+Unfortunately we still have the restriction that _all physical servers need to be on the
```

Comment:
Off-topic, but because you're working on this, can you confirm / refute the following? https://github.com/dedis/onet/issues/671